### DISCUSSION TITLE
Question or feat. request.  Switch to previous used window/column. / Alt+Tab window switcher addon?

### DISCUSSION THREAD

**Discussion by megatux** [YaLTeR/niri#292] [Ideas]
I'm too used to Alt+Tab to switch to my recents windows/apps.
Is there something like a queue of the recently used windows or at least the previous one, so I can bind a key to quickly switch to it?

Also, I remember using something like that with i3...maybe from this [Redit post](https://www.reddit.com/r/i3wm/comments/95b2hx/alttab_like_behavior/) to get a better idea of what I mean.

**Comment by YaLTeR** [comment]
Actually sfwbar can do it, you bind Alt-Tab in niri to sending SIGUSR1 to sfwbar or something
